app/templates/login.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+     <div class="container">
+
+            <h1>Sign In</h1>
```

Comment:
The current name of the tab is Login. Thought on updating one or the other to for consistency?

Reply:
I think I've changed everything over to Log In or login.